CHANGELOG.md

```diff
@@ -2,6 +2,22 @@
 
 ## Unreleased
 
+## v0.66.0
+
+This Splunk OpenTelemetry Collector release includes changes from the [opentelemetry-collector v0.65.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.65.0), which has the same content as [opentelemetry-collector v0.66.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.66.0), the [opentelemetry-collector-contrib v0.65.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.65.0), and the [opentelemetry-collector-contrib v0.66.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.66.0) releases where appropriate.
+
+### 💡 Enhancements 💡
+
+- Add alpha `k8sobjects` receiver [#2270](https://github.com/signalfx/splunk-otel-collector/pull/2270)
+- Add Windows 2022 Docker image support [#2269](https://github.com/signalfx/splunk-otel-collector/pull/2269)
+- Remove usage of collector experimental code in config sources [#2267](https://github.com/signalfx/splunk-otel-collector/pull/2267)
+- Update internal ConfigSource to follow upstream confmap.Provider implementation [#2271](https://github.com/signalfx/splunk-otel-collector/pull/2271)
```

Comment:
I would recommend not including these two since they are just refactoring of internal packages and don't affect end users in any way

Reply:
I considered not, but given https://github.com/signalfx/splunk-otel-collector/pull/2274 I think it's good to signal something has changed in this area so if unforeseen issues occur there's something to point to in our changelog.

Reduced to a single item since they are internal.